docs/Toy/toy/dialect.py

```diff
@@ -157,10 +157,10 @@ def verify_(self):
 
         block = self.body.blocks[0]
 
-        if not len(block.ops):
+        if block.is_empty:
```

Comment:
For a few reasons, first is that it is an O(1) check, second that it is stateless, third is that it's not supposed to change in between mutating operations

Reply:
Full disclosure: also because I'm used to it being a property: https://developer.apple.com/documentation/swift/collection/isempty-35uy0 It's a property in Swift for the same reasons